test/e2e/util/utils.go

```diff
@@ -213,6 +213,46 @@ func parseRepoTag(img string) (RepoTag, error) {
 	return RepoTag{}, err
 }
 
+func HelmDeployLatestEraserRelease(namespace string, extraArgs ...string) env.Func {
+	return func(ctx context.Context, cfg *envconf.Config) (context.Context, error) {
+		addEraserRepoCmd := exec.Command("bash", "-ec", `
+        helm repo add eraser https://azure.github.io/eraser/charts
```

Comment:
can we make the "https://azure.github.io/eraser/charts" a const or var in case this changes later on?

Reply:
done